newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js

```diff
@@ -314,15 +317,46 @@ export default class PlatformSpecificAssetsDialog extends React.Component<
         onApply={this.onApply}
       >
         <ColumnStackLayout noMargin>
+          <Line justifyContent="center" noMargin>
+            {isResizeSupported() ? (
+              <ElementWithMenu
+                element={
+                  <RaisedButton
```

Comment:
I'd use a RaisedButtonWithSplitMenu here too

Reply:
Reverted back to a simple button :)